### ISSUE TITLE
Warn when attachment was forgotten

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/mail#1179]
When you mention the words »attachment« or »attached« (or similar ones) in the email and there is no attachment on the message, ownCloud Mail should ask if you forgot the attachment.

Established apps do that and it’s a nice way to avoid awkward mails.

**Comment by ChristophWurst** [comment]
When implementing that feature, we should also localize it, e.g. look for "Anhang" if the user uses the mail app in German.

**Comment by jancborchardt** [comment]
Yeah, I guess simply putting the keywords up for translation will already do the trick.

**Comment by aspdye** [comment]
I think we should create a List ... In German there is not only "Anhang" "anbei" or "angehängt" ... I think we should put them all in all tenses and ... .
For over 100 Languages this Could Get an big Project 😜

**Comment by jancborchardt** [comment]
Well, the point is to start with the few words which will solve the majority of use-cases. : ) You know »the perfect is the enemy of the good«, eh?
